content/manuals/security/for-admins/provisioning/_index.md

```diff
@@ -38,6 +38,12 @@ When a user signs in through SSO, Docker obtains several attributes from your Id
 - **Docker Org**: Optional. Specifies the organization the user belongs to
 - **Docker Team**: Optional. Defines the team the user belongs to within the organization
 - **Docker Role**: Optional. Determines the user's permission within Docker
+- **Docker session minutes**: Optional. Determine's the user's session limit.
```

Comment:
thought: IMO, "you can set session ..from the IDP .. " sounds a bit redundant, why not instead specify that the user will need to authenticate again with their IDP after these minutes have passed since they signed in